clients/py/sapphirepy/tests/test_e2e.py

```diff
@@ -20,9 +20,12 @@
 def compiled_test_contract():
     if not os.path.exists(GREETER_ABI):
         # pylint: disable=import-outside-toplevel
-        from solcx import compile_source    # type: ignore
+        import solcx   # type: ignore
+
         with open(GREETER_SOL, 'r', encoding='utf-8') as handle:
-            compiled_sol = compile_source(
+            solcx.install_solc(version='0.8.9')
+            solcx.set_solc_version('0.8.9')
+            compiled_sol = solcx.compile_source(
                 handle.read(),
                 output_values=['abi', 'bin']
```

Comment:
Might be possible to add `evm_version='paris'`
Then can remove `set_solc_version`, and use `install_solc('latest')`?

Reply:
Sure, will add in a separate PR.